src/systems/filecoin_blockchain/storage_power_consensus/_index.md

```diff
@@ -44,54 +46,124 @@ More precisely,
+{{<label tickets>}}
+## Tickets
 
-In Filecoin, every block header contains a single ticket.
+Filecoin block headers also contain a single "ticket" generated from its epoch's beacon entry. Tickets are used to break ties in the Fork Choice Rule, for forks of equal weight.
 
 You can find the Ticket data structure {{<sref data_structures "here">}}.
 
-### Comparing Tickets in a Tipset and using ticket values
-
-Whenever comparing tickets is evoked in Filecoin, for instance when discussing selecting the "min ticket" in a Tipset, the comparison is that of the ticket's VRFDigest's bytes.
-
-Likewise any use of a ticket's value, is that of its VRFDigest's bytes.
-
-{{<label ticket_chain>}}
-## The Ticket chain and drawing randomness
-
-While each Filecoin block header contains a ticket field (see {{<sref tickets>}}), it is useful to think of a ticket chain abstraction.
-Due to the nature of Filecoin's Tipsets and the possibility of using tickets from epochs that did not yield leaders to produce randomness at a given epoch, tracking the canonical ticket of a subchain at a given height can be arduous to reason about in terms of blocks. To that end, it is helpful to create a ticket chain abstraction made up of only those tickets to be used for randomness generation at a given height.
-
-To read more about specifically how tickets are processed for randomness, see {{<sref randomness>}}.
+Whenever comparing tickets in Filecoin, the comparison is that of the ticket's VRFDigest's bytes.
 
 {{<label ticket_generation>}}
 ### Randomness Ticket generation
 
-This section discusses how tickets are generated by EC for the `Ticket` field in every block header.
+At a Filecoin epoch n, a new ticket is generated using the appropriate beacon entry for epoch n.
 
-At round `N`, a new ticket is generated using tickets drawn from the Tipset at round `N-1` (as shown below).
-
-The miner runs the prior ticket through a Verifiable Random Function (VRF) to get a new unique ticket which can later be derived for randomness (as shown above). The prior ticket is prepended with the ticket domain separation tag and concatenated with the miner actor address (to ensure miners using the same worker keys get different randomness).
+The miner runs the beacon entry through a Verifiable Random Function (VRF) to get a new unique ticket. The beacon entry is prepended with the ticket domain separation tag and concatenated with the miner actor address (to ensure miners using the same worker keys get different tickets).
 
 To generate a ticket for a given epoch n:
 ```text
-LastTicket = MinTicketValueAtEpoch(n-1)
-newRandomness = VRF_miner(H(TicketProdDST || index || Serialization(pastTicket, minerActorAddress)))
+randSeed = GetRandomnessForEpoch(n)
+newRandomness = VRF_miner(H(TicketProdDST || index || Serialization(randSeed, minerActorAddress)))
```

Comment:
We have different randomnesses in Filecoin. We should name them all individually. This should be ElectionProofRandomness (correct?)

Reply:
not correct, this is for ticket production (for the vrf chain). You can tell from the DST. But overall randomness seeds are all taken using the same method (assuming you meant we should name the method individually), which seems to me to be the right level of abstraction. Thereafter, the randomness only changes as it's assembled (using the dst, serialization, etc)... If you meant name the var, indeed feel free to change it!

Reply:
name of the var.